asciidoctorj-api/src/main/java/org/asciidoctor/ast/ContentPart.java

```diff
@@ -0,0 +1,27 @@
+package org.asciidoctor.ast;
+
+import java.util.List;
+import java.util.Map;
+
+public interface ContentPart {
+
+    public String getId();
+
+    public int getLevel();
+
+    public String getContext();
+
+    public String getStyle();
+
+    public String getRole();
+
+    public String getTitle();
+
+    public Map<String, Object> getAttributes();
+
+    public String getContent();
+
+    public List<ContentPart> getParts();
+
+    public void setParts(List<ContentPart> parts);
```

Comment:
I do not think that `setParts(List<ContentPart>)` should be on the API. The only usage is in [JRubyAsciidoctor line 182](https://github.com/jmini/asciidoctorj/blob/patch-1/asciidoctorj-core/src/main/java/org/asciidoctor/internal/JRubyAsciidoctor.java#L182) in `JRubyAsciidoctor.getContentPartFromBlock(StructuralNode, int, int)`. At this point the setter on `ContentPartImpl` is enough. 

This would be more consistent with the other method on this interface.

Reply:
:+1: